man/adoc/bpftrace.adoc

```diff
@@ -2865,6 +2870,21 @@ END {
 }
 ----
 
+[#probes-bpftrace]
+=== bpftrace
+
+.variants
+* `bpftrace:signal:SIGUSR1`
+
+These are special built-in events provided by the bpftrace runtime.
+They are triggered when the bpftrace process receives specific events, such as a `SIGUSR1` signal.
```

Comment:
does this need more words about how this overrides the built in behaviour (of printing maps) and how the bpf prog trigger is indirct (i.e. it depends on the runtime calling a trigger function). Also, what happens if you define two of these probes?